src/main/steps/applicant1/check-contact-details/content.ts

```diff
@@ -0,0 +1,35 @@
+import { TranslationFn } from '../../../app/controller/GetController';
+
+const en = () => ({
+  title: 'Review your contact details',
+  yourAddress: 'Your address',
+  yourPhoneNumber: 'Your phone number',
+});
+
+// @TODO translations
+const cy = en;
+
+const languages = {
+  en,
+  cy,
+};
+
+export const generateContent: TranslationFn = content => {
+  const address = [content.formState?.applicant1Address1];
+  address.push(content.formState?.applicant1Address2);
```

Comment:
why is formState using ?. isn't it always set?

Reply:
i believe its the way we do generateContent, as we are passing in a partial<caseData> its complaining it might be empty.

Reply:
if the formState is partial then then ?. should be on applicant1Address2 but you're doing an undefined check on formState

Reply:
oh, looking into it, its also a case where formstate is optional in commonContent function, looks like for the error controller as it dont need it there to be passed, we can turn that off and pass it in.

Reply:
I would always set it even if it's not needed. Saves using ?. everywhere

Reply:
kl, ill do that tho i think that should be another pr, changing it here will make this pr even bigger.

Reply:
sure, do it in another PR